pkg/odo/cli/version/version.go

```diff
@@ -46,6 +49,13 @@ func NewVersionOptions() *VersionOptions {
 
 // Complete completes VersionOptions after they have been created
 func (o *VersionOptions) Complete(name string, cmd *cobra.Command, args []string) (err error) {
+	if !o.clientFlag {
+		// Let's fetch the info about the server
+		o.serverInfo, err = genericclioptions.ClientWithConnectionCheck(cmd, true).GetServerVersion()
```

Comment:
Won't this fail if the connection to the server can't be made?

With the code in `master`, `odo version` does not fail when the connection to the server cannot be made. It just doesn't print the server info

Reply:
Hmm, I didn't think of that, let me check.